### DISCUSSION TITLE
Включать/выключать вид tree

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#460] [Как это сделать?] [answered]
Скажите пожалуйста, есть ли возможность в системе как пользователь включать/выключать вид деревом (если нет, то как можно это реализовать)? Иногда удобно искать что-то в дереве, а иногда просто видеть списком.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
никто за это никогда не заплатит
